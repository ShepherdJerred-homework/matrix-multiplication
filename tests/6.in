5 355 748 440 773 185 488 608 493 426 62 22 573 824 820 775 790 196 105 202 123 25 590 458 702 607 210 587 970 657 492 147 564 550 249 559 13 701 775 510 489 598 951 938 538 446 90 967 757 568 635  609 645 992 234 821
