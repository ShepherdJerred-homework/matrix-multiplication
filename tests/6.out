1467902
1328994
2028546
859941
1512726
2026415
1516108
1749671
2400131
2083716
